packages/component-submission/client/pages/NewSubmissionWizard.js

```diff
@@ -78,7 +84,12 @@ const NewSubmissionWizard = ({ initialValues, match, history }) => {
                 />
                 <TrackedRoute
                   path={`${match.path}/disclosure`}
-                  render={() => <DisclosureStep {...formikProps} />}
+                  render={() => (
+                    <DisclosureStep
+                      isSubmissionAttempted={submissionAttempted}
```

Comment:
Maybe we can move the error message up to this level so that it doesn't sit in the Disclosure step? This would make the form more easily customisable (removing the disclosure page wouldn't require moving the whole form validation message)

Reply:
The error message is part of the disclosure page and not inside the modal so not sure where we would put it as it wouldn't render anywhere else.